ui/app/AppLayouts/Communities/views/EditCommunityTokenView.qml

```diff
@@ -66,6 +68,9 @@ StatusScrollView {
 
         readonly property int imageSelectorRectWidth: root.isAssetView ? 128 : 290
 
+        readonly property bool containsAssetReferenceName: root.isAssetView ? SQUtils.ModelUtils.contains(root.referenceAssetsBySymbolModel, "name", nameInput.text) : false
+        readonly property bool containsAssetReferenceSymbol: root.isAssetView ? SQUtils.ModelUtils.contains(root.referenceAssetsBySymbolModel, "symbol", symbolInput.text) : false
+
```

Comment:
It could be done like that or via SFPM with two `ValueFilter`'s inside `AnyOf` filter. Then it prevents from duplicates a bit better. `ModelUtils.contains` checks only in time of invocation, it won't be reevaluated when `referenceAssetsBySymbolModel` changes internally.

So let's say that user provides input, it's checked that's ok, but then in some community we know sb has created token with given name or symbol. This won't be detected in current approach and action of creating token with such duplicated name/symbol won't be blocked. On the other hand the the approach with SFPM is able to detect that so I would opt for using that approach here.

Reply:
Good point. Going to update it!